examples/with-firebase/fetchData/getProfileData.js

```diff
@@ -0,0 +1,15 @@
+import admin from '../firebase/nodeApp'
+
+export const gettProfileData = async (username) => {
+  const db = admin.firestore()
+  const profileCollection = db.collection('profile')
+  const profileDoc = await profileCollection.doc(username).get()
+
+  if (!profileDoc.exists) {
+    return null
+  }
```

Comment:
How is the user created for the example?

Reply:
I've added a button on the index page to register as a user.